### PR TITLE
Migrate friendly names

### PR DESCRIPTION
**Pull request by howonlee**
Bad things happen without the explicit data migration, turns out

### CODE REVIEW

**Review by dpsutton — CHANGES_REQUESTED** [review]
This should go into the schema migrations file `resources/migrations/000_migrations.yml`.

We really want to stay away from these data migrations when at all possible. there's a couple issues with them but the biggest issue (IMO) is that they run after all of the other schema migrations. So conceivably in the future we run migrations for 0.56 in the schema migrations and then this one for 0.40 would run _after_. 

`UPDATE setting set value = 'simple' where key = 'humanization-strategy' and value = 'advanced';`

**Comment by howonlee** [comment]
I thought you couldn't do data migrations in liquibase but this turned out to be another migration thing or I was misremembering. Willfix

**Review by dpsutton — APPROVED** [review]
Looks good to me. Have you verified this on h2, pg, and mysql? Only thing i'd be worried about is how portable the where clause is between the three, although it looks good to me.

**Comment by howonlee** [comment]
There was a material bug in mysql which I whacked

File: resources/migrations/000_migrations.yaml [hunk 1]
```diff
@@ -8369,6 +8369,21 @@ databaseChangeLog:
...
+      author: howonlee
+      comment: Added 0.40.0 Migrate friendly field names
+      changes:
+        - update:
+            tableName: setting
+            columns:
+              - column:
+                  name: value
+                  value: 'simple'
+            where: "'key'='humanization-strategy' AND value='advanced'"
```
Comment (camsaul) [inline resources/migrations/000_migrations.yaml:8385]: I think this is broken

```sql
WHERE 'key' = 'humanization-strategy'
```

is comparing two string literals that will never be equal. This should have been `key`, not `'key'`

Comment (flamber) [reply resources/migrations/000_migrations.yaml:8385]: It is wrong, and was fixed in changeset 311 right after #16482

Comment (camsaul) [reply resources/migrations/000_migrations.yaml:8385]: Oh right, forgot about that. We probably should have removed the old one or at least added a comment to it at the time.

Let's do that now so it doesn't trip any more people up

**Pull request merged**